recipes/glib/all/conanfile.py

```diff
@@ -43,6 +43,8 @@ def _is_msvc(self):
     def validate(self):
         if hasattr(self, 'settings_build') and tools.cross_building(self, skip_x64_x86=True):
             raise ConanInvalidConfiguration("Cross-building not implemented")
+        if tools.Version(self.version) >= "2.69.0" and not self.options.with_pcre:
+            raise ConanInvalidConfiguration("option glib:with_pcre must be True for glib >= 2.69.0")
```

Comment:
what does it mean? Why not delete the option instead?

Reply:
the error message is clearer like this

Reply:
But this option doesn't exist anymore in glib >= 2.69.0, correct? Therefore it's confusing to keep it.

Reply:
in the long term sure.
For now, this allows consumers having glib:with_pcre=True to build flawlessly

Reply:
It's a limitation of conan client, making recipes confusing. AFAIK, `glib:with_pcre=True` would work in command line even if with_pcre option is deleted, but it would hard fail if it's specified in a conanfile.